### ISSUE TITLE
BoH interfiers with swarmer teleportation

### ISSUE THREAD

**Issue by HomocidlePancakes** [tgstation/tgstation#44233]
![nich](https://user-images.githubusercontent.com/35217076/58705714-b119f400-83b8-11e9-95f7-a8a1e0498cd5.png)

And that may end up teleporting you in to lava

**Comment by GunHog** [comment]
I would not consider this a bug, but a risk inherent with wearing the item. Any benign teleportation would be affected by the BoH to become dangerous. Imagine the RD wearing his tele armor and a BoH, this is rather common. Simply being shot by something, say, a swarmer, would teleport him, and the BoH would extend the range of that teleportation, possibly spacing him or worse. Besides, Lavaland swarmers could teleport you next to a hungry Goliath. 

TL;DR: You choose to accept this risk when you wear the BoH. Any and all teleports, including ones meant to be "safe", become a risk. It is the price you pay for having the best storage item in the game!

**Comment by Tlaltecuhtli** [comment]
how did you end in lavaland with a swarmer

**Comment by HomocidlePancakes** [comment]
*shrug. You sure allowing swarmers to kill people is a good idea?

**Comment by Dorsisdwarf** [comment]
The BoH is _supposed_ to make even benign and safe teleportation dangerous. Don't wear one if there's swarmers about, it aint their fault if you end up in fucksville.
